### PR TITLE
Add Enterprise ListRunners/RegistrationToken APIs

### PR DESCRIPTION
**Pull request by jamie-34254**

### CODE REVIEW

**Comment by jamie-34254** [comment]
@googlebot I signed it!

**Comment by jamie-34254** [comment]
We just signed the corporate CLA, so I'm guessing there might be a delay on it?

**Comment by gmlewis** [comment]
@googlebot is usually very fast. There might be an issue. Let's see if it responds with an update.

**Comment by jamie-34254** [comment]
@googlebot I signed it!

**Comment by gmlewis** [comment]
> We just signed the corporate CLA, so I'm guessing there might be a delay on it?

Oh, I gotcha... I read that too quickly before.  Yes, there is a manual review process for brand new corporate CLAs... so we may have to wait a few days.

Thank you for your patience, @jamie-34254 !

**Comment by jamie-34254** [comment]
@googlebot I signed it!

File: github/enterprise_actions_runners.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import (
+	"context"
+	"fmt"
+)
+
+// CreateRegistrationToken creates a token that can be used to add a self-hosted runner.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/actions/#create-a-registration-token-for-a-repository
+func (s *EnterpriseService) CreateRegistrationToken(ctx context.Context, enterprise string) (*RegistrationToken, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners/registration-token", enterprise)
```
Comment (gmlewis) [inline github/enterprise_actions_runners.go:17]: When I follow the link on line 15, I see an endpoint like this:

```
POST /repos/{owner}/{repo}/actions/runners/registration-token
```

not the endpoint I see on line 17.  Why is that?

File: github/enterprise_actions_runners.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	}
+
+	return registrationToken, resp, nil
+}
+
+// ListRunners lists all the self-hosted runners for a enterprise.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/enterprise-admin/#list-self-hosted-runner-groups-for-an-enterprise
+func (s *EnterpriseService) ListRunners(ctx context.Context, enterprise string, opts *ListOptions) (*Runners, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners", enterprise)
```
Comment (gmlewis) [inline github/enterprise_actions_runners.go:37]: There seems to be a mismatch here too.  The documentation link on line 35 points to an endpoint that looks like this:

```
GET /enterprises/{enterprise}/actions/runner-groups
```

and not the one listed on line 37.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @jamie-34254 - sorry for the delay, I didn't get a notification that the CLA issue had been resolved, unfortunately.

I have a couple questions if you could please clear them up for me.

**Review by gmlewis — APPROVED** [review]
Thank you, @jamie-34254 !
LGTM.

Once you fix the CLA problems (see comment about your email address), then this will be ready for a second LGTM and merging.

**Comment by jamie-34254** [comment]
Thanks @gmlewis! This should be good for 2nd review now :)

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**